unity-ggjj/Assets/Scripts/AudioController/AudioController.cs

```diff
@@ -1,9 +1,11 @@
 using System.Collections;
 using UnityEngine;
-using UnityEngine.Serialization;
 
 public class AudioController : MonoBehaviour, IAudioController
 {
+    [Tooltip("Drag a NarrativeScriptPlaylist here")]
```

Comment:
Nit: Mismatching tooltip text and property class.
```suggestion
    [Tooltip("Drag a DialogueController here")]
```